modules/shared/src/main/scala/org/tessellation/shared/package.scala

```diff
@@ -72,7 +72,7 @@ package object shared {
     classOf[SnapshotTips] -> 607,
     classOf[ActiveTip] -> 608,
     ActiveTip.OrderingInstance.getClass -> 609,
-    classOf[BlockAsActiveTip[DAGBlock]] -> 610,
+    classOf[BlockAsActiveTip[_]] -> 610,
```

Comment:
If it works then it means that `BlockAsActiveTip[CurrencyBlock]` is not needed. I remmeber that Kryo didn't care about the parameter of generic so good that it works.

Reply:
Yeah afaik the type parameter here is ignored anyway and the id for `DAGBlock` (or for `CurrencyBlock`) is taken from the registry. I also tested it on the cluster and checked that all unit test work.